src/coreclr/src/System.Private.CoreLib/src/System/DateTime.Windows.CoreCLR.cs

```diff
@@ -11,15 +11,15 @@ namespace System
         private static extern unsafe bool ValidateSystemTime(Interop.Kernel32.SYSTEMTIME* time, bool localTime);
 
         [MethodImpl(MethodImplOptions.InternalCall)]
-        private static extern unsafe bool FileTimeToSystemTime(long fileTime, FullSystemTime* time);
+        private static extern unsafe bool FileTimeToSystemTime(ulong fileTime, FullSystemTime* time);
```

Comment:
It may be nice to delete these and switch to fully managed implementation as done in the other PR to avoid the need for this dual-maintenance.

Reply:
But the other PR is going to supersede this part anyway? Or is that still far off?